coil-base/src/main/java/coil/memory/MemoryCache.kt

```diff
@@ -78,6 +77,38 @@ interface MemoryCache {
         override fun toString(): String {
             return "Key(key=$key, extras=$extras)"
         }
+
+        override fun writeToParcel(parcel: Parcel, flags: Int) {
+            parcel.writeString(key)
+            parcel.writeInt(extras.size)
+            extras.forEach {
+                parcel.writeString(it.key)
+                parcel.writeString(it.value)
+            }
+        }
+
+        override fun describeContents(): Int {
+            return 0
+        }
+
+        companion object CREATOR : Parcelable.Creator<Key> {
```

Comment:
Let's make this just an `object` instead of `companion object` since we don't want to support all the syntax that comes with a `companion object`.